uk.co.compucorp.civicrm.hrcore/CRM/HRCore/CMSData/Role/DrupalRoleService.php

```diff
@@ -21,19 +15,39 @@ public function getRoleNames() {
    * @inheritdoc
    */
```

Comment:
Please indicate this this function throws an exception.

Reply:
I've asked the other devs about this because I'm not sure. Personally I always turn off this warning in PHPStorm since I don't think it adds any value, but it will be good to hear other opinions